components/Breadcrumb/src/breadcrumb.css

```diff
@@ -0,0 +1,125 @@
+    var(--denhaag-color-grey-2)
+  );
+}
+
+.denhaag-breadcrumb__link {
+  color: var(--denhaag-breadcrumb-link-color, var(--denhaag-link-color));
+  padding-block-start: var(--denhaag-breadcrumb-spacing, var(--denhaag-space-inline-xs));
+  padding-block-end: var(--denhaag-breadcrumb-spacing, var(--denhaag-space-inline-xs));
+  padding-inline-start: var(--denhaag-breadcrumb-spacing, var(--denhaag-space-inline-xs));
+  padding-inline-end: var(--denhaag-breadcrumb-spacing, var(--denhaag-space-inline-xs));
+  pointer-events: none;
+  position: relative;
+}
+
+.denhaag-breadcrumb__link:hover,
+.denhaag-breadcrumb__link--hover {
+  --denhaag-breadcrumb-link-color: var(--denhaag-breadcrumb-link-hover-color, var(--denhaag-link-hover-color));
+}
+
+.denhaag-breadcrumb__link:focus,
+.denhaag-breadcrumb__link--focus {
+  --denhaag-breadcrumb-link-color: var(--denhaag-breadcrumb-link-focus-color, var(--denhaag-link-focus-color));
+  outline: var(--denhaag-breadcrumb-link-focus-outline, var(--denhaag-link-focus-outline));
+}
+
+.denhaag-breadcrumb__link--current,
+.denhaag-breadcrumb__link.denhaag-link--current,
+.denhaag-breadcrumb__item:last-child > .denhaag-breadcrumb__link {
+  --denhaag-breadcrumb-link-color: var(--denhaag-breadcrumb-current-color, var(--denhaag-breadcrumb-color));
+  text-decoration: none;
+}
+
+.denhaag-breadcrumb__link:not(.denhaag-breadcrumb__link--current):not(.denhaag-breadcrumb__link.denhaag-link--current):not(:last-child
+    > .denhaag-breadcrumb__link)
+  > :first-child {
+  pointer-events: bounding-box;
+}
+
+/* SVG for the separator */
+.denhaag-breadcrumb__link > .denhaag-breadcrumb__text + .denhaag-icon {
+  color: initial;
+  font-size: inherit;
+  margin-inline-start: var(--denhaag-breadcrumb-spacing, var(--denhaag-space-inline-xs));
+  margin-inline-end: calc(-1 * var(--denhaag-breadcrumb-spacing, var(--denhaag-space-inline-xs)));
+}
+
+[dir="rtl"] .denhaag-breadcrumb__link > .denhaag-breadcrumb__text + .denhaag-icon {
+  transform: scaleX(-1);
+}
+
+.denhaag-breadcrumb__text {
+  background: var(--denhaag-breadcrumb-background-color);
+  overflow: hidden;
+  position: relative;
+  text-overflow: ellipsis;
+  transition: background var(--denhaag-breadcrumb-transition-ease) var(--denhaag-breadcrumb-transition-speed);
+  white-space: nowrap;
+}
+
+.denhaag-breadcrumb__link > .denhaag-icon + .denhaag-breadcrumb__text {
+  /* Copied from Bootstrap .sr-only */
+  border: 0;
+  clip: rect(0, 0, 0, 0);
+  height: 1px;
+  margin-inline-start: -1px;
+  margin-inline-end: -1px;
+  margin-block-start: -1px;
+  margin-block-end: -1px;
+  overflow: hidden;
+  padding-block-start: 0;
+  padding-block-end: 0;
+  padding-inline-start: 0;
+  padding-inline-end: 0;
+  position: absolute;
+  width: 1px;
+  white-space: nowrap;
+}
+
+.denhaag-breadcrumb__item:not(:first-child):not(:nth-last-child(2)):not(:last-child) .denhaag-breadcrumb__text {
+  aspect-ratio: 24/16;
+  border-radius: var(--denhaag-breadcrumb-dots-border-radius, 3px);
```

Comment:
There is a default border radius token for the entire library. `--denhaag-border-radius`. That might be a better fallback value, since it allows users to define only the generic `--denhaag-border-radius` to have a consistent border radius style across all components.

Reply:
Ahh! Thankgod, I was looking for a more generic value, but couldn't find it. I stumbled upon the `--denhaag-card-border-radius` variant and some others which i didn't want to use. I will apply this right away.